### PR TITLE
Improve link a11y for downloads, repeating links

### PR DESCRIPTION
**Pull request by mrnosal**
# Summary
Improve accessibility by adding additional information to download links and disambiguating repetitive links on the Test Kit and Resources pages. 

# Testing Guidance
Confirm links are valid and are differentiated via title text (Test Kit page)

### CODE REVIEW

File: community/resources.md [hunk 1]
```diff
@@ -15,25 +16,30 @@ US Core Test Kit Tutorial: [https://github.com/inferno-training/inferno-tutorial
...
-- [ONC 2022 Tech Forum](https://www.healthit.gov/news/events/2022-onc-virtual-tech-forum): Inferno Framework Capabilities Beyond (g)(10) - [Link to Video](https://www.youtube.com/watch?v=epf7OHPaeZ0)
+- [HL7 Synthetic Data Event](https://confluence.hl7.org/pages/viewpage.action?pageId=184922592): Synthetic Data for Testing<br>Video: [HL7 Synthetic Data Event - August 29, 2023](https://youtu.be/WcMfL4tQEOQ?si=hJWOt13Ouo2JLQW5&t=2723)
+- [ONC 2022 Tech Forum](https://www.healthit.gov/news/events/2022-onc-virtual-tech-forum): Inferno Framework Capabilities Beyond (g)(10)<br>Video: [ONC 2022 Tech Forum: Inferno Framework Capabilities Beyond (g)(10)](https://www.youtube.com/watch?v=epf7OHPaeZ0)
+{:itemlist}
 
 ## Inferno Architecture Diagram
 
-You can view an [Inferno Architecture Diagram](/download/Inferno_Architecture.pdf) (PDF) which summarizes the components of Inferno Framework and includes a glossary of terms.
+You can view an [Inferno Architecture Diagram (PDF, 1.1MB)](/download/Inferno_Architecture.pdf) (PDF) which summarizes the components of Inferno Framework and includes a glossary of terms.
```
Comment (AlyssaWang) [inline community/resources.md:45]:
```suggestion
You can view an [Inferno Architecture Diagram (PDF, 1.1MB)](/download/Inferno_Architecture.pdf) which summarizes the components of Inferno Framework and includes a glossary of terms.
```

File: docs/concepts.md [hunk 2]
```diff
@@ -105,4 +105,4 @@ a RESTful API to enable third-party integration.
 
 ## Inferno Architecture Diagram
 
-You can view an [Inferno Architecture Diagram](/download/Inferno_Architecture.pdf) (PDF) which summarizes the components of Inferno Framework and includes a glossary of terms.
+You can view an [Inferno Architecture Diagram (PDF 1.1 MB)](/download/Inferno_Architecture.pdf) which summarizes the components of Inferno Framework and includes a glossary of terms.
```
Comment (AlyssaWang) [inline docs/concepts.md:108]: Maybe we should be linking this in a separate page if we reference it more than once across pages. Thoughts?

**Review by AlyssaWang — COMMENTED** [review]
Links generally work although I didn't bother testing all the table ones. Will approve after suggestions are reviewed.

File: docs/concepts.md (hunk 2 above)
Comment (mrnosal) [reply docs/concepts.md:108]: I think it's fine to have the links on both pages as is - it isn't changing frequently, and is trivial to move to a partial if needed.

File: assets/common.css [hunk 3]
```diff
@@ -54,6 +54,10 @@ pre[class*=language-] {
 .news-content {
   margin-left: 1.5rem;
 }
+/* Lists */
+.item-list li {
+  margin-bottom:1em;
+}
```
[the remark this answers is not in the document]
Comment (mrnosal) [reply assets/common.css]: thx - updated.

File: community/resources.md (hunk 1 above)
Comment (mrnosal) [reply community/resources.md]: Thx - removed extraneous (PDF)

**Review by AlyssaWang — APPROVED** [review]

**Pull request merged**